examples/methods/list.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
Can you also add examples for:
- https://docs.mollie.com/orders/migrate-to-orders#retrieving-payment-methods
- https://docs.mollie.com/payments/multicurrency#filtering-payment-methods

Reply:
This client doesn't support Orders (yet). I'll add the filtering example though 👍